### DISCUSSION TITLE
teamviewer alternative

### DISCUSSION THREAD

**Discussion by info100t** [amidaware/tacticalrmm#1126] [Ideas]
Hi,

I discover your project and wow... great job !

Because i can't this year, next year if money come back, i'll sponsors you.

I'm not software programer, i'm network admin so i can only have idea ! 📦 

Suggest : teamviewer integration alternative... https://www.supremocontrol.com/fr/

Next year, for money reason, i'll leave teamviewer to join and use supremo. (teamviewer = 2500$, supremo = 300$).

Romain

**Comment by silversword411** [comment]
Looks like it was already on here...with the other options ;)
https://docs.google.com/spreadsheets/d/1Z5JUC-FCYAIzyW2rAoXLpQYq0a8VFJ5VUh5gqSerWPY/

**Comment by info100t** [comment]
wtf !! :)

I don't know if someone ask the question to supremo support..  So because i'm a new supremo reseller, I just sent them an email to ask them the question ;)

**Reply by silversword411** [reply]
If they want to writeup integration documentation similar to https://docs.tacticalrmm.com/3rdparty_screenconnect/

I'll be happy to integrate it into the docs

**Comment by info100t** [comment]
ok ;)

fyi, i update the spreadsheets with supremo price.

Sorry it's in euro ! :p 

romain

**Comment by info100t** [comment]
hi !

hahaha... it's same than teamviewer to get id !!! I don't understand all the script but.... it work ! :)

I try and i come back to you.

```
# Retrieve Supremo ID from TRMM agent.

$SupremoVersionsNums = @('4', '')
$RegPaths = @('HKLM:\SOFTWARE\Wow6432Node\Supremo')
$Paths = @(foreach ($SupremoVersionsNum in $SupremoVersionsNums) {
        foreach ($RegPath in $RegPaths) {
            $RegPath + $SupremoVersionsNum
        }
    })

foreach ($Path in $Paths) {
    If (Test-Path $Path) {
        $GoodPath = $Path
    }
}

foreach ($FullPath in $GoodPath) {
    If ($null -ne (Get-Item -Path $FullPath).GetValue('ClientID')) {
        $SupremoID = (Get-Item -Path $FullPath).GetValue('ClientID')
        $ErrorActionPreference = 'silentlycontinue'
    }
}

Write-Output $SupremoID

Exit $LASTEXITCODE
```

**Comment by info100t** [comment]
So it work...

Now to lauch Supremo with id, i need to do C:\XXXX\supremo.exe ID

When i know how can we do that, i'll can do a documentation :)

![image](https://user-images.githubusercontent.com/105393872/168348386-ddb07692-2294-4a20-b94c-6fdb9cdfa735.png)

**Reply by dinger1986** [reply]
URL Action - https://www.supremocontrol.com/tutorials/how-to-start-a-connection-with-supremo-using-a-link-or-url/#:~:text=Integrating%20Supremo%20in%20third%2Dparty,the%20password
